### ISSUE TITLE
Get chords

### ISSUE THREAD

**Issue by dezudas** [martijnversluis/ChordSheetJS#1446]
How to Use getChords()

it is not working for me

**Comment by martijnversluis** [comment]
Hey @dezudas. hanks for reaching out. It should be fixed now. There was a bug in the release build process. 20.4.1 should have the getChords() included.

**Comment by dezudas** [comment]
it worked but gives me duplicate chords 
```
const initialChord = `[Title: Song Title]
[Artist: Leonard Cohen]

[Intro]

E          C#m    G#m         A
Sambodhan timilai gardai chu ma
E      C#m     G#m     A
Sikeka ti sabai gun harule
E       C#m     G#m       A
Samarpan geet yo gardai chu
E      C#m      G#m       A
Timrai ti mitha raag haru le
G#m   A           E
Chadi gayau hami lai
G#m      A       E
Jivan ko yo modma
G#m      A          E
Samjhana bhai bilayau
     Bm             D
Hawa sangai udi gayou
   Bm       D      E
Yaada matra bana yo

[Verse 1]

E       C#m    G#m    A
Samjhana mutuma aairancha
E         C#m    G#m       A
Bitholcha biyog ka palharu le
E     C#m     G#m     A
Tadiyau timi nabhetne gari
E        C#m     G#m    A
Angalcha timrai bhawana le
G#m    A           E
Chadi gayau hami lai
G#m      A       E
Jivan ko yo modma
G#m      A          E
Samjhana bhai bilayau
     Bm             D
Hawa sangai udi gayou
   Bm       D      E
Yaada matra bana yo

[Verse 2]

E          C#m    G#m         A
Sambodhan timilai gardai chu ma
E     C#m    G#m     A
Sikeka ti sabai gun harule
E       C#m     G#m       A
Samarpan geet yo gardai chu
E      C#m      G#m       A
Timrai ti mitha raag haru le`;

  const parseAndSetSong = (chordSheet) => {
    try {
      const parsedSong = new ChordSheetJS.UltimateGuitarParser().parse(chordSheet);
      if (parsedSong) {
        setSong(parsedSong);
        setCifra(formatSongToHtml(parsedSong));
        setUniqueChords(parsedSong.getChords());
      } else {
        console.error("Failed to parse song.");
      }
    } catch (error) {
      console.error("Error parsing song:", error);
    }
  };
  ```
  [codesandbox](https://codesandbox.io/p/sandbox/parse-chords-hnpwv6?file=%2Fsrc%2FApp.js%3A73%2C1-94%2C5)

**Comment by martijnversluis** [comment]
Hey @dezudas. You're right, the function did not account for whitespace or subtle difference in notation of the same chord. 

I made some changes and published `10.7.1` which should improve `getChords()`. Please let me know if this works for you 👍

**Comment by dezudas** [comment]
@martijnversluis no changes in the result still can see the duplicate content 

Used Chords: E C#m G#m AE C#m G#m E G#m C#m G#m A EG#m A A Bm D Bm D G#m E E E

[codesandbox](https://codesandbox.io/p/sandbox/parse-chords-hnpwv6?file=%2Fsrc%2FApp.js%3A33%2C20)

**Comment by martijnversluis** [comment]
Hey @dezudas. Thanks for checking. As it turns out, Parcel's caching caused most updates since `10.7.0` not to appear in the compiled bundle. I generated and published version `10.9.2` after disabling the cache. Could you confirm that version is working correctly?

**Comment by dezudas** [comment]
@martijnversluis  it works perfect 

https://codesandbox.io/p/sandbox/parse-chords-hnpwv6?file=%2Fsrc%2FApp.js%3A187%2C16
